### ISSUE TITLE
Questions about the division of data sets into test sets and training sets

### ISSUE THREAD

**Issue by freeLGR** [YU-deep/CRISP_SAM2#5]
Thank you for your excellent work. I have a question.I would like to know how you divide the test set of the amos dataset. The amos test set does not have a public GT file. Do you use the validation set as the test set?How do you divide the amos dataset? I would be grateful if you could answer.
